package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nordcloud/gnui",
   "description": "Nordcloud Design System - a collection of reusable React components used in Nordcloud's SaaS products",
-  "version": "2.8.0",
+  "version": "3.0.0",
```

Comment:
Are there any breaking changes? Or publishing only `dist` directory is considered one?